release/artifacts.yaml

```diff
@@ -33,15 +33,19 @@
   type: jar-scala
 - target: //daml-lf/data-scalacheck:data-scalacheck
   type: jar-scala
+- target: //daml-lf/encoder:encoder
+  type: jar-scala
```

Comment:
I am a bit worried we start releasing the parser and the encoder. 
1. the quality of those libraries is not so good (in particular parsing error are pretty bad)
2  the syntax handle by the parser is evolved in a non backward compatible way. 
3. their is not proper documentation

Currently, the language team does not have the resources to improve any of this 3 point.

If the code handle by the parser stays in the main repo, the language team will be able to fix it as the syntax as the parser evolved.

Reply:
As discussed offline, I'm going to try to avoid publishing the parser and the encoder by building the required DAR files beforehand.